src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java

```diff
@@ -395,8 +395,15 @@ public void writeCsvResponse(Result[] resultSet, HttpServletResponse servletResp
           .filter(t -> keysInt.contains(t.getKey())).toArray(OSHDBTag[]::new));
     }
     if (includeOSMMetadata) {
-      properties =
-          addAdditionalProperties(entity, properties, isContributionsEndpoint, contributionTypes);
+      properties.put("@version", entity.getVersion());
+      properties.put("@osmType", entity.getType());
+      properties.put("@changesetId", entity.getChangesetId());
+      if (isContributionsEndpoint) {
+      properties = addContributionTypes(properties, contributionTypes);
+    }
```

Comment:
aahm… what happened to the indentation here? :see_no_evil: PS: please don't just tick off the _My code follows the code-style rules_ checkbox without actually checking it :stuck_out_tongue:

Reply:
oops, there were quite a few of them all over the place here. See https://jenkins.ohsome.org/job/ohsome-api/job/master/508/checkstyle/new/. I've opened https://github.com/GIScience/ohsome-api/pull/173 to hopefully fix all of them.

PS: please install and use the checkstyle plugin in your [IDE](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#code-style) and make it a habit to run it befor submitting a PR. :pray: 

PSS: please consider configuring your IDE to remove superfluous trailing whitespace at the end of lines, like [below](https://github.com/GIScience/ohsome-api/pull/162/files#diff-9175ccbae42fea93a094d76cb63b8f7cc0f5a58ea80a48615cd5d57ce6f3dc6eR142) ([fix](https://github.com/GIScience/ohsome-api/pull/173/files#diff-9175ccbae42fea93a094d76cb63b8f7cc0f5a58ea80a48615cd5d57ce6f3dc6eL142-R142)).